pages/documentation/documentation.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="styles.css" />
+		<title>Documentation | UI Game</title>
+	</head>
+	<body class="main-body">
+		<header class="body-header">
+			<h1 class="margin-right">UI Game</h1>
+			<a href="https://github.com/zoef-yt/ui-game"><h3>Github Repo</h3></a>
+		</header>
+
+		<main class="body-content"></main>
+
+		<footer class="body-footer-1">Made with ❤️ zoef shaikh</footer>
```

Comment:
You can put a div to contain both footer and apply common styling on the container. **just my opinion**.

Reply:
This is an extra file that was included in the commit it will be removed in the next commit.

Reply:
2 footers are used for grids.